### PR TITLE
[PAY-668] Set up premium content

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Depends on https://github.com/AudiusProject/audius-protocol/pull/3928 (now merged) and new libs version release (also done)

- Consume libs premium content changes
- Add premium content fields to track upload metadata, setting the premiumness to false by default. We have no UI for making tracks premium as of now.

### Dragons

Danger here is potentially borking the track upload flow. Testing the upload was fine but extra scrutiny on that piece is welcome.

### How Has This Been Tested?

Local client against remote dev box. Was able to upload and play tracks.

Testing track edits...
Update: track edit works!

### How will this change be monitored?

No monitoring here really, only that tracks have new is_premium and premium_conditions fields. Requesting tracks from DNs returns those fields as well.

### Feature Flags ###

No feature flags necessary as it does not introduce new UI. Overall behavior should remain the same.

### CODE REVIEW

**Comment by sddioulde** [comment]
edit track no longer erroring, but not able to confirm either. getting to the bottom of that...

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc

File: packages/web/src/pages/upload-page/UploadPage.js [hunk 1]
```diff
@@ -262,6 +262,7 @@ class Upload extends Component {
       this.state.stems
     )
     this.changePage(Pages.FINISH)
+    this.setState({ tracks })
```
Comment (rickyrombo) [inline packages/web/src/pages/upload-page/UploadPage.js:265]: For my own context, what happened/why was this needed?

File: packages/web/src/common/store/player/sagas.ts [hunk 2]
```diff
@@ -24,7 +26,13 @@ import {
   delay
 } from 'typed-redux-saga'
 
+import { waitForLibsInit } from 'services/audius-backend/eagerLoadUtils'
+
 import errorSagas from './errorSagas'
+
+// @ts-ignore
+const libs = (): AudiusLibs => window.audiusLibs
```
Comment (rickyrombo) [inline packages/web/src/common/store/player/sagas.ts:34]: We can't include libs in common sagas direcly afaik, maybe put the necessary functionality in AudiusBackend and use `getContext('audiusBackendInstance)`? cc: @dylanjeffers maybe

**Review by rickyrombo — CHANGES_REQUESTED** [review]
Looks pretty good! I think we have to refrain from keeping libs in common sagas directly since common is used in mobile.

File: packages/web/src/common/store/player/sagas.ts (hunk 2 above)
Comment (rickyrombo) [reply packages/web/src/common/store/player/sagas.ts:34]: wonder if we can't put a linter on common to check for `window` usages? Although since we have the `@ts-ignore` we'd probably miss this anyway

Comment (dylanjeffers) [reply packages/web/src/common/store/player/sagas.ts:34]: yeah the move is to grab them from context, happy to show you how, just dm

File: packages/web/src/pages/upload-page/UploadPage.js (hunk 1 above)
Comment (sddioulde) [reply packages/web/src/pages/upload-page/UploadPage.js:265]: i dont think we need this actually.
wanted to be consistent with the state before i merged this small client PR https://github.com/AudiusProject/audius-client/pull/2042/files

but pretty sure this does not really do anything once we switch to the FINISH page.
going to simply remove and make sure things still work

File: packages/web/src/common/store/player/sagas.ts (hunk 2 above)
Comment (sddioulde) [reply packages/web/src/common/store/player/sagas.ts]: got it! thx. please re-review if latest change makes sense :)

**Review by rickyrombo — APPROVED** [review]
